charts/osm/templates/osm-deployment.yaml

```diff
@@ -73,6 +73,33 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.name
+      {{- if .Values.OpenServiceMesh.enableFluentbit}}
+        - name: fluentbit-logger
+          image: fluent/fluent-bit:1.5
```

Comment:
Should we also add this as a parameter in values.yaml?

Reply:
by this do you mean the name of the sidecar?

Reply:
I'm referring to the image name and tag. Similar to how users can configure the values for the osm controller image by changing the values in the values.yaml:
`image: "{{ .Values.OpenServiceMesh.image.registry }}/osm-controller:{{ .Values.OpenServiceMesh.image.tag }}"`

Reply:
ah yes, that makes sense